packages/vue-components/src/index.js

```diff
@@ -59,8 +56,8 @@ const components = {
   popover,
   tooltip,
   modal,
-  BPopover,
-  BTooltip,
+  'VPopover': Dropdown,
+  'VTooltip': Tooltip,
```

Comment:
Correct me if I'm wrong but should these be under directives instead?

Reply:
floating-vue does provide a [`v-tooltip` directive](https://floating-vue.starpad.dev/guide/directive.html), however the ones imported here are components ([`Menu`](https://floating-vue.starpad.dev/guide/component.html#hover-menu) and [`Tooltip`](https://floating-vue.starpad.dev/guide/component.html#tooltip)) which are components :)

Reply:
Ahh, got confused when looking at those haha.